test/commands/resources/create-issue-input.valid.yaml

```diff
@@ -0,0 +1,59 @@
+sellerName: test
+sellerMarketPlaceAlias: cloufsoft-developer
+accountId: "012345678901"
+productName: myapp
+productCategory: networking
+productUrl: https://test.io/software
+addon:
+  name: myapp
+  versionName: test myapp 0.1.0 mp1
+  version: 1.0.0
+  namespace: test-myapp
+  type: networking
+  helmChartUrl: 709825985650.dkr.ecr.us-east-1.amazonaws.com/test-development/test-myapp-helm:0.1.6
```

Comment:
What AWS Account number is this? Lets remove an AWS Account numbers in public repos.

Reply:
Ack, it was the cloudsoft acc number; will swap it for a fake one

Reply:
I missed this sorry. 
The account `709825985650` is the marketplace account used for creating the ECR repositories for all the add-ons.

When a repository is added to a product, the ISV only provide the suffix, and the repo is created under that account on `ecr.us-east-1` with the ISV alias as prefix eg: `709825985650.dkr.ecr.us-east-1.amazonaws.com/isv-marketplace-alias/repo-name-provided`

Adding this comment for documentation. I don't think is needed to put back the account to the examples.